### ISSUE TITLE
In RA portal it should only be possible to assign RA(A) rights to users with an active LoA3 registration

### ISSUE THREAD

**Issue by phavekes** [OpenConext/Stepup-RA#369]
[This issue is imported from pivotal - Originaly created at Oct 4, 2016 by Eefje van der Harst](https://www.pivotaltracker.com/story/show/131617617)

In RA portal it should only be possible to assign RA(A) rights to users with an active LoA3 registration. 

At the moment it is also possible to assign these rights to users with an active LoA2 registration. You can then assign RA(A) rights, but the user gets an access denied when logging in to RA_portal. Not very user-friendly. Better to prevent this from happening by adding an extra LoA-check before assigning extra rights to users.

Blocked by #133173121

Est.: 16hrs.

**Comment by phavekes** [comment]
Instead of querying and showing a feedback page, we could just filter the list of users that can be accredited. (Alex Rothuis - Oct 31, 2016)

**Comment by phavekes** [comment]
It is difficult to get the LoA information necessary for filtering the RaCandidates (or checking their LoA after requesting accreditation) as this information is currently not part of the RaCandidate projection. There is no stable way of doing this without introducing new projections and/or events. (Alex Rothuis - Nov 3, 2016)

**Comment by phavekes** [comment]
Simple solution:
- only show the RaCandidates with a second factor with LoA3 in the search/overview pages.

Before estimation:
- verify that a mapping between second factor type and LoA3 is manageable and configurable enough to ensure that this is possible. Think about the usecase with multiple LoA3 second factors. (Daan van Renterghem - Jan 23, 2017)

**Comment by phavekes** [comment]
For now we avoid introducing the SAML concept of LoA into the middleware (Daan van Renterghem - Jan 25, 2017)

**Comment by phavekes** [comment]
Relevant mapping: https://github.com/SURFnet/Stepup-bundle/blob/develop/src/Value/SecondFactorType.php
 (Daan van Renterghem - Jan 25, 2017)
